http_server_test.go

```diff
@@ -554,18 +547,6 @@ var registerEndpointTests = map[string]struct {
 		status:          200,
 		json:            tinkerbellDataModel,
 	},
-	"default endpoint": {
-		url:    "/metadata",
-		status: 200,
-		json:   tinkerbellDataModel,
-	},
-	"custom endpoints invalid format (not a map)": {
-		customEndpoints: `"/userdata":".metadata.userdata"`,
-		url:             "/userdata",
-		status:          404,
-		error:           "error in parsing custom endpoints: invalid character ':' after top-level value",
-		json:            tinkerbellDataModel,
-	},
```

Comment:
what's the reasoning for removing these two cases?   
shouldn't we have a case that tests the default? and the second one is for covering the bases of all the possible invalid envvar formats

Reply:
I had originally done the CUSTOM_ENDPOINTS parsing in `func main()` and so removed it. But I didn't want to affect codecoverage too much so put it behind the func we have now. But I forgot to bring this test back in. Will fix.

Reply:
fixed, PTAL. I left the default one out as thats done in `func main()` now, as it should be :D.